virttest/qemu_vm.py

```diff
@@ -1269,6 +1276,8 @@ def add_balloon(devices, devid=None, bus=None, use_old_format=None):
             dev = QDevice(model, parent_bus=bus)
             if devid:
                 dev.set_param("id", devid)
+            for k in kwargs:
+                dev.set_param(k, kwargs[k])
```

Comment:
How about following format:
```
for key, value in kwargs.items():
    dev.set_param(key, value)
```

Reply:
Yes, this could work too. But I'm not sure which one is better or both are ok.